### ISSUE TITLE
construct adjacency matrix from training set

### ISSUE THREAD

**Issue by pursueorigin** [hwwang55/KGNN-LS#2]
Hi,

Your code is very clear. But I have a question for the construction of the adjacency matrix. Because you split the data set into train, eval, and test set, the adjacency matrix should be constructed from the training set in the training phase, rather than the entire dataset. Is it correct?  In my opinion, the recommendation task is actually a link prediction task. When training the model, we should not include links which are in the testing/validation set. 

Thanks.
